src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -75,7 +75,7 @@
     </ItemGroup>
     <PropertyGroup>
       <PathToMergedNuGetPack>$(ILMergeResultDir)\NuGet.Build.Tasks.Pack.dll</PathToMergedNuGetPack>
-      <DotnetSharedDirectoryArg Condition="'$(TargetFramework)' == 'netstandard1.3'">/lib:$(OutputPath)$(RuntimeIdentifier)\publish\refs</DotnetSharedDirectoryArg>
+      <DotnetSharedDirectoryArg Condition="'$(TargetFramework)' == 'netstandard1.3'">/lib:$(DotnetSharedSDKDirectory)</DotnetSharedDirectoryArg>
```

Comment:
This should use the IS_CORECLR (sp?) property instead of netstandard1.3, which we could have other netstandard frameworks in the future.